Project-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js

```diff
@@ -1,5 +1,8 @@
 import React from 'react';
 import './Card.css';
+import CardBanner from './CardBanner';
+import CardContent from './CardContent';
```

Comment:
If you'd like and this is just a personal preference, but you could use a React Fragment to help your code and jsx syntax to run faster than usual!